lib/atomic-management.js

```diff
@@ -116,21 +116,16 @@ export default {
       this.askReload();
     }
 
-    // If a package is listed as disabled but not installed, ask user if they want to install it
     const installedPackageNames = new Set(
       atom.packages.getAvailablePackageNames()
     );
-    const notInstalledPackageNames = newDisabled.filter(
-      packageName => !installedPackageNames.has(packageName)
-    );
-    if ('*' in contents) {
-      for (var toplevel in contents) {
-        for (var pname in contents[toplevel]) {
-          if (
-            pname != 'core' &&
-            pname != 'editor' &&
-            !installedPackageNames.has(pname)
-          ) {
+    const notInstalledPackageNames = [];//newDisabled.filter(packageName => !installedPackageNames.has(packageName));
+    if("*" in contents){
+      for(var toplevel in contents){
```

Comment:
Can we change the varaible name from `toplevel` to `scope`. I think it will help clarify what is going on here. Cause these toplevels are just defining scoped settings. Let me know what you think